Sources/TuistAutomation/Device/DeviceController.swift

```diff
@@ -0,0 +1,168 @@
+import Command
+import FileSystem
+import Foundation
+import Mockable
+import Path
+import TuistSupport
+import XcodeGraph
+
+enum DeviceControllerError: FatalError {
+    case applicationVerificationFailed
+
+    var description: String {
+        switch self {
+        case .applicationVerificationFailed:
+            "The app could not be installed because the verification failed. Make sure that your device is registered in your Apple Developer account."
```

Comment:
What happens when the app is registered but the device is not included in the provisioning profile of the app? Is it the same error or a different one?

Reply:
It's this one, yeah.